src/Node.java

```diff
@@ -29,4 +29,11 @@ public ArrayList<Edge> getEdges() {
     public void addEdge(Edge edge){
         edges.add(edge);
     }
+
+    @Override
+    public String toString() {
+        return "Node{" +
+                "value=" + value +
+                '}';
```

Comment:
It'd be useful to also include the isFinal and at least the number of edges. Something like `a(3).` would be great to visualize a node with value `a`, having 3 edges and being final (the dot).